src/robot/libraries/BuiltIn.py

```diff
@@ -834,22 +917,33 @@ def should_contain(self, container, item, msg=None, values=True):
         | Should Contain | ${output}    | PASS  |
         | Should Contain | ${some list} | value |
         """
+        if is_truthy(ignore_case) and is_string(container) and is_string(item):
+            container = container.lower()
+            item = item.lower()
         if item not in container:
             raise AssertionError(self._get_string_msg(container, item, msg,
                                                       values, 'does not contain'))
 
-    def should_contain_x_times(self, item1, item2, count, msg=None):
+    def should_contain_x_times(self, item1, item2, count, msg=None, ignore_case=False):
         """Fails if ``item1`` does not contain ``item2`` ``count`` times.
 
         Works with strings, lists and all objects that `Get Count` works
         with. The default error message can be overridden with ``msg`` and
         the actual count is always logged.
 
+        ``ignore_case`` is False by default.  If True, it indicates that
+        ``item1`` and ``item2`` should be compared case-insensitively,
+        provided that ``item1`` and ``item2`` are string types.  See `Boolean
```

Comment:
Same discussion about docs and implementation as with `should_not_contain` applies also to this keyword.